labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,123 @@
+var userPlayer;
+var grant;
+
+window.addEventListener("load", () => {
+	document.getElementById('start-button').addEventListener('click', startGame);
+	document.getElementById('attack-button').addEventListener('click', attack);
+	document.getElementById('heal-button').addEventListener('click', heal);
+	document.getElementById('quit-button').addEventListener('click', quit);
+});
+
+function startGame() {
+	const playersName = document.getElementById('players-name').value;
+	userPlayer = new Player(playersName, 40);
+	grant = new Player('Grant', 10);
+	document.getElementById('start-div').style.display = 'none';
+	document.getElementById('main-div').style.display = 'flex';
+	document.getElementById('users-name').innerText = playersName;
+	document.getElementById('grants-name').innerText = 'Grant The Almighty Chicken';
+	document.getElementById('heals-remaining').innerText = 'Heals Left: ' + userPlayer.numberOfHeals + '/2';
+	document.getElementById('win-count').innerText = 'Wins: ' + userPlayer.wins + '/5'
+}
+
+function attack() {
```

Comment:
Woah! This function is MASSIVE and is doing a LOT. 
- Do the actual attack
- What color should the health bars be? 
- Set the message for both players! 
- Test who won

One Thing™ per function, please! That way you can name it and everyone can know what it's supposed to do.